src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -119,6 +120,17 @@
     boolean isChatColorsEnabled();
 
     /**
+     * Simulates a chat message from a player.
+     *
+     * <p>This method sends a message as if it came from this player.
+     * It does not send a message to this player.</p>
+     *
+     * @param message The message to send
+     * @return The event that was thrown from sending the message
+     */
+    MessageChannelEvent.Chat chat(Text message);
```

Comment:
> It does not send a message to this player.

I'm not quite sure about this one. Because it implies that the used MessageChannel explicitly excludes the given player. Although the player would normally get a formatted version of what he just typed.